lib/runners/processor/clang_tidy.rb

```diff
@@ -0,0 +1,106 @@
+module Runners
+  class Processor::ClangTidy < Processor
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!({
+          apt: enum?(string, array(string)),
+          'compilation-options': object?(
+            '-I': enum?(string, array(string))
+          )
+        })
+      }
+
+      let :issue, object(
+        severity: string,
+      )
+    end
+
+    register_config_schema(name: :clang_tidy, schema: Schema.runner_config)
+
+    VALID_EXTENSIONS = [".c", ".cc", ".cpp", ".c++", ".cp", ".cxx"].freeze
+    GLOB_HEADERS = "**/*.{h,hpp,h++}".freeze
+
+    def analyze(changes)
+      deploy_packages
+      run_analyzer(changes)
+    end
+
+    private
+
+    def deploy_packages
+      # select "lib*" and report others as warning for security concerns
+      packages = Array(config_linter[:apt])
+        .select do |pkg|
+          if pkg.start_with?("lib")
+            true
+          else
+            add_warning "Installing the package '%s' is blocked." % pkg
+            false
+          end
+        end
+
+      unless packages.empty?
+        capture3!("sudo", "apt-get", "install", "-y", "--no-install-recommends", *packages)
```

Comment:
[suggestion] How about suppress the verbose log as follows?

```suggestion
        capture3!("sudo", "apt-get", "install", "-qq", "-y", "--no-install-recommends", *packages)
```

Reply:
Good catch!
commit 44b237bda22e18255686d5cf0abd5f184386548c